src/geoFire/interfaces/geoQueryCallbacks.ts

```diff
@@ -0,0 +1,6 @@
+export interface GeoQueryCallbacks {
+  ready: any[];
```

Comment:
In `src/geoFire/interfaces/geoQueryState.ts` we have this interface:

```typescript
(a: firebase.database.DataSnapshot, b?: string) => any
```

If this is the callback type could we name it and instead of an `any[]` use that?

_Ditto for the 3 props below_